lib/active_shipping/carrier.rb

```diff
@@ -67,6 +68,19 @@ def create_shipment(origin, destination, packages, options = {})
       raise NotImplementedError, "#create_shipment is not supported by #{self.class.name}."
     end
 
+    # Cancels a shipment with a carrier.
+    #
+    # @note Override with whatever you need to cancel a shipping label
+    #
+    # @param shipment_id [String] The unique identifier of the shipment to cancel. 
+    # This can be shipment_id or tracking number depending on carrier. Up to you and 
+    # the carrier
+    # @return [ActiveShipping::ShipmentResponse] The response from the carrier. This
+    #   response in most cases has a cancellation id.
+    def cancel_shipment(shipment_id)
```

Comment:
Following with the existing pattern of the interface methods can you add a second argument `options = {}`